core/src/main/java/org/arquillian/cube/impl/client/container/ContainerConfigurationController.java

```diff
@@ -64,15 +64,16 @@ public void remapContainer(@Observes BeforeSetup event, CubeRegistry cubeRegistr
         Object newConfigurationInstance = configurationClass.newInstance();
 
         for (PropertyDescriptor configurationClassPortField : configurationClassPortFields) {
+            // Move it outside if condition below, if there is a group of containers.
+            int containerPort = getDefaultPortFromConfigurationInstance(newConfigurationInstance,
```

Comment:
The error will persist if the it doesn't enter on this for loop.
Does it make sense to move it above the for? If not you'll have to repeat the assignment and the conditional inside the second for.